docusaurus/docs/cloud/getting-started/deployment.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Deployment
+displayed_sidebar: cloudSidebar
+description: Learn how to deploy your Strapi application on Strapi Cloud.
+canonicalUrl: https://docs.strapi.io/cloud/getting-started/deployment.html
+---
+
+# Strapi Cloud
+
+This is a step-by-step guide for deploying your Strapi application on Strapi Cloud. Before you are able to access Strapi Cloud, you must first [request access](#requesting-access) to the Beta release.
+
+## Prerequisites
+
+Before you can deploy your Strapi application on Strapi Cloud, you need to have the following prerequisites:
+
+* Strapi version `4.6.x` or higher
```

Comment:
Update here: I just caught up with @thenewnkem and he confirmed that we'll need to minimally require Strapi version `4.7.x` instead of `4.6.x`.